### DISCUSSION TITLE
Integrating rainbow wallet using Wagmi

### DISCUSSION THREAD

**Discussion by technitish9123** [rainbow-me/rainbowkit#764] [Q&A]
I have to integrate rainbow wallet using wagmi . I don'nt wish to integrate using rainbow kit instead i want to integrate it using InjectedConnector of wagmi.
Please help me out.

**Comment by peduarte** [comment]
Hey there, if you'd like to create your own connect wallet functionality without using RainbowKit, you can refer to the [wagmi docs](https://wagmi.sh/docs/getting-started). they also have a great [example here](https://wagmi.sh/examples/connect-wallet).

Note that if you want to add support for Rainbow, you need to use the [WalletConnect Connector](https://wagmi.sh/docs/connectors/walletConnect)
